source/color_raw_to_rgb.ipynb

```diff
@@ -0,0 +1,326 @@
+{
```

Comment:
<p>##  Get the data into kornia by doing the conversion </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kornia/tutorials/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>